CHANGELOG.md

```diff
@@ -111,6 +111,7 @@
 - [Named arguments syntax is now recognized in IDE][5774]. Connections to
   function arguments will now use named argument syntax instead of inserting
   wildcards on all preceding arguments.
+- [Added boilerplate React app for authorization via Cognito+AWS Amplify][5798].
```

Comment:
Please make it a little bit more wordy and easier to read. The Changeling should be understandable by no-code users :)